test/testdata/HlintNoRefactorings.hs

```diff
@@ -1,4 +0,0 @@
-main = putStrLn "hello"
```

Comment:
wow, i didnt notice those hlint tests and i wrote equivalent ones from zero 🤦

Reply:
Good reason to clean things up: so we can see what's there. :)